cmd/soroban-rpc/internal/db/transaction.go

```diff
@@ -339,9 +282,6 @@ func newTransactionTableMigration(ctx context.Context, logger *log.Entry, retent
 		}
 		// Truncate the table, since it may contain data, causing insert conflicts later on.
 		// (the migration was shipped after the actual transactions table change)
-		// FIXME: this can be simply replaced by an upper limit in the ledgers to migrate
```

Comment:
has this FIXME been addressed?

Reply:
@2opremio and I decided to remove this FIXME. I will remove it in this PR.